internals.Rmd

```diff
@@ -2,70 +2,558 @@
 source("common.R")
 ```
 
-# ggplot2 internals {#internals}
+```{r echo=FALSE}
+# Function that mimics ggplot_build(), but instead of returning the 
```

Comment:
I think this is big enough it would be better to do in its own file (which you'd then `source()` here), and it needs a few comments at the top describing how you derived it from ggplot2 sources (so if we come back to this for the fourth edition, we have some collective memory of how it was created)

Reply:
Done. It's probably worth checking to see if my comments in the sourced files are sufficient though!